src/bundle/check.ts

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+import path from "path";
+
+const searchInFile = (filePath: string, searchString: string) => {
+  const content = fs.readFileSync(filePath, "utf8");
+  return content.includes(searchString);
+};
+
+export const searchInDirectory = (dirPath: string, searchString: string) => {
+  const files = fs.readdirSync(dirPath);
+  let found = false;
+
+  files.forEach((file) => {
```

Comment:
This will iterate over all files in the package. Using [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) will break the loop once a file with the search string was found.